meta/state.go

```diff
@@ -342,7 +343,18 @@ func (r *localRaft) leader() string {
 		return ""
 	}
 
-	return r.raft.Leader()
+	leader := r.raft.Leader()
+	// TODO remove debug code
```

Comment:
no, that's why I have a TODO in there.  This is debugging code as I only see these failures on circle and can't reproduce them locally.